### ISSUE TITLE
Shelly 1PM Mini Gen3 Support

### ISSUE THREAD

**Issue by juanfal** [alexryd/homebridge-shelly-ng#126]
[**Shelly 1PM Mini Gen3**](https://www.shelly.com/en/products/shop/shelly-1-pm-mini-gen3) is not supported.  It is a great upgrade of the previous, and really cheap!, but impossible to add to _HomeBridge_

**Comment by masav61** [comment]
Did you try the forked repo [homebridge-shelly-ds9](https://github.com/cubi1337/homebridge-shelly-ds9)?
This should support Gen3 devices and it worked for me.
Thanks to @apartmedia

**Comment by juanfal** [comment]
Yes!  I finally have managed to add it to HK and, at the same time, see it through the Shelly app, that gives a quite complete info and statistics about energy consume. Great

**Comment by SilvioMigu** [comment]
hello everybody
I use the HOOBS platform
How can I integrate this "homebridge-shelly-ds9" with HOOBS ?
Can someone explain it to me?
thanks

**Comment by SilvioMigu** [comment]
I find this plugin
Is that ?
![image](https://github.com/user-attachments/assets/15cf928b-af75-4ca7-b60b-b40669d51bc4)

**Comment by masav61** [comment]
Hi Vitor,

Sorry but I don’t know the HOOBS platform.

Marcello
Il 30 ott 2024, 22:16 +0100, Vitor ***@***.***>, ha scritto:
